src/main/java/com/spotinst/sdkjava/model/bl/gcp/OceanGKETaints.java

```diff
@@ -0,0 +1,88 @@
+package com.spotinst.sdkjava.model.bl.gcp;
```

Comment:
File location and name to be changed as the general comment.

Reply:
Resolved